src/test/kotlin/uk/gov/justice/digital/hmpps/courtlistsplitter/messaging/SqsMessageReceiverIntTest.kt

```diff
@@ -32,11 +43,37 @@ class SqsMessageReceiverIntTest {
   @Autowired
   private lateinit var telemetryService: TelemetryService
 
+  @Autowired
+  private lateinit var messageNotifier: MessageNotifier
+
+  @Autowired
+  private lateinit var amazonSQSAsync: AmazonSQSAsync
+
+  @AfterAll
+  internal fun beforeAll() {
+    System.err.println("SHUT DOWN")
```

Comment:
Is this needed for readability or leftover code from debugging or something?

Reply:
Oh - missed that ! Done.